src/module/modal.js

```diff
@@ -0,0 +1,89 @@
+import commentApi from './commentApi.js';
+
+const modalMethods = {
+  async show(id, src, mealName, category) {
+    // const meals = await getMeals(url);
+
+    const modal = document.querySelector('#staticBackdrop');
+    modal.innerHTML = '';
+    const allContent = document.createElement('div');
+    allContent.classList.add('modal-dialog');
+    allContent.innerHTML = `
+      <div class="modal-content">
+        <div class="modal-header">
+          <h2 class="modal-title" id="staticBackdropLabel">${mealName}</h2>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        </div>
```

Comment:
- [ ] When the comments popup loads, the web app should retrieve data from the selected API and should show details about the selected item. At the moment, you are only retrieving data from the Involvement to show the item comments